backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/weather/WeatherController.java

```diff
@@ -0,0 +1,30 @@
+package de.neuefische.ffmjava221.teamprojekt.backend.weather;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/api/weather")
+public class WeatherController {
+
+    private final WeatherService weatherService;
+
+    public WeatherController(WeatherService weatherService) {
+        this.weatherService = weatherService;
+    }
+
+    @GetMapping("/{date}")
+    public WeatherData getWeather(@PathVariable String date, @RequestParam int hour) {
```

Comment:
I would probalby not transmit the date as a Pathvariable and the time as a requestparameter... Im not sure how your api works. But maybe you could send it combined as a Date-Time Object (Instant?) or maybe seperated as two Request Parameters...